engine/src/main/resources/templates/Common/insightsEmailBody.html

```diff
@@ -0,0 +1,521 @@
+                                                    </p>
+                                                </td>
+                                                <td class="rh-masthead__section__head">
+                                                    <p><span class="rh-masthead__section__head__counter">{#insert content-title-right-part-section2}{/}</span>
+                                                </td>
+                                            </tr>
+                                        </table>
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td class="rh-content__block" style="padding: 0px 0px 24px 0px;"/>
+                                </tr>
+                                <tr>
+                                    <td class="rh-content__block">
+                                        {#insert content-body-section2}{/}
+                                    </td>
+                                </tr>
+                                {#if renderButtonSection2}
+                                <tr>
+                                    <td class="rh-content__block" style="padding: 24px 0px;">
+                                        <table align="center">
+                                            <tr>
+                                                <td class="rh-cta-link" align="center">
+                                                    {#insert content-button-section2}{/}
+                                                </td>
+                                            </tr>
+                                        </table>
+                                    </td>
+                                </tr>
+                                {#else}
+                                <tr>
+                                    <td class="rh-content__block" style="padding: 0px 0px 24px 0px;"/>
+                                </tr>
+                                {/if}
+                            </table>
+                        </td>
+                    </tr>
+                    {/if}
+
+                    <!-- end body section -->
+
+                    <!-- Footer -->
+                    <tr>
+                        <td class="rh-footer">
+                            <table role="presentation" border="0" align="center" cellpadding="0" cellspacing="0" width="100%">
+                                <tr>
+                                    <td class="rh-footer__text">
+                                        This email was sent by Hybrid Cloud Console | <a href="{environment.url}/user-preferences/notifications/{action.bundle}" target="_blank">Manage email preferences</a>
+                                    </td>
+                                </tr>
+                            </table>
+                        </td>
+                    </tr>
+                    <!-- end footer -->
+                </table>
```

Comment:
The old template contained an empty table row with a `don't remove` comment. Is it really okay to remove it? 😃

Reply:
We don't need padding bottom according UXD: https://docs.google.com/document/d/17Lba2pepEnbem5LLdLxtqPPp_22Bjo38PQVLUXX5vCQ/edit

Reply:
We don't need padding bottom according UXD